web/src/features/charts/elements/AreaGraph.tsx

```diff
@@ -76,6 +76,8 @@ const getLayers = (
   }));
 };
 
+const defaultFormatTickHandler = (t: number) => t.toString();
+
```

Comment:
Cant we simply use String() instead?

If I remember correctly it should do the same.

Reply:
In this case, where we don't accept any null-ish inputs it's probably fine to use String() :)
Updated now.